.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @i-am-bee/ui-developers
```

Comment:
Is the team publicly visible and has write access? I am getting this error:

```Unknown owner on line 1: make sure the team @i-am-bee/ui-developers exists, is publicly visible, and has write access to the repository```

Reply:
Me too, I'm not sure, that's why I added also @matoushavlena as a reviewer. But I guess we'll see, if it works, once enabled.